tests/test_transformDiaSourceCatalog.py

```diff
@@ -105,7 +105,8 @@ def test_run(self):
         # The final snr value should be NaN because it doesn't have a peak significance field.
         expect_snr = [397.887353515625]*9
         expect_snr.append(np.nan)
-        self.assertTrue(np.array_equal(result.diaSourceTable["snr"], expect_snr, equal_nan=True))
+        # Have to use allclose because assert_array_equal doesn't support equal_nan.
+        np.testing.assert_allclose(result.diaSourceTable["snr"], expect_snr, equal_nan=True, rtol=0)
```

Comment:
I'm not sure the comment is necessary -- since the transformation seems to be doing unit or convention conversions (see comments above about significance vs. SNR), isn't it unsafe to assume exact equality anyway?

Reply:
I believe in this case exact equality is what we want: there should be no numeric change in this value between the source catalog and the pandas table.

Reply:
What do you mean by "see comments above about significance vs. SNR"? I didn't see a comment from you related to those two.

Reply:
I meant the pre-existing comments in the code. 🙂 

But I'm not very familiar with what transform actually does, so if you say there are no conversions, I'll take your word for it.